pkgs/tools/inputmethods/netevent/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchFromGitHub, docutils }:
+
+stdenv.mkDerivation rec {
+  pname = "netevent";
+  version = "20201018";
+
+  src = fetchFromGitHub {
+    owner = "Blub";
+    repo = "netevent";
+    rev = "ddd330f0dc956a95a111c58ad10546071058e4c1";
+    sha256 = "0myk91pmim0m51h4b8hplkbxvns0icvfmv0401r0hw8md828nh5c";
+  };
+
+  buildInputs = [ docutils ];
+
+  outputs = [ "out" "doc" "man" ];
+
+  RST2MAN = "rst2man";
```

Comment:
It looks like this isn't necessary, netevent already uses rst2man from PATH under this name

Reply:
When not setting RST2MAN to this value, for some reason, the configure (which is not a classical configure) does not find the binary. This ensure the check is skipped and rst2man is effectively used.

Reply:
Please export this in configure.